src/app/datapoints-graph/charts/echarts-options.service.ts

```diff
@@ -527,10 +527,13 @@ export class EchartsOptionsService {
     value += `<li class="p-t-4 p-b-4 d-flex separator-bottom text-no-wrap"><label class="text-label-small m-b-0 m-r-8">Last Updated</label><span class="small m-l-auto">${this.datePipe.transform(alarm['lastUpdated'])}</span></li>`;
     const exists = await this.alarmRouteExists();
     if (exists) {
-      const currentUrl = this.router.url;
-      const baseUrlIndex = currentUrl.indexOf('/index.html#');
-      const baseUrl = currentUrl.substring(0, baseUrlIndex + 11);
-      value += `<li class="p-t-4 p-b-4 d-flex separator-bottom text-no-wrap"><label class="text-label-small m-b-0 m-r-8">Link</label><span class="small m-l-auto"><a href = "${baseUrl}/alarms/${alarm.id}">Alarm Details</a></span></li>`;
+      const currentUrl = window.location.href;
+      const baseUrlIndex = currentUrl.indexOf('/index.html');
+      const baseUrl = currentUrl.substring(
+        0,
+        baseUrlIndex + '/index.html'.length
```

Comment:
I would suggest moving `/index.html` into a variable instead of having it here twice..

Reply:
done